inst/sensorreading/sensorreading.Rmd

```diff
@@ -62,8 +62,11 @@ if(!identical(tbl, "The dataset requested is empty.")){
 ***
 
 ```{r,echo=FALSE,result='asis',comment=NA,warning=FALSE}
+library(dplyr)
```

Comment:
So, it looks like I need this (or something like it). I kind of wonder if I can put it somewhere else though. Does the importFrom line actually import anything or is it just used for documentation?